src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/metadata/ProjectSampleMetadataController.java

```diff
@@ -165,8 +165,16 @@ public SampleMetadataStorage createProjectSampleMetadata(HttpSession session, @P
 		Iterator<Cell> headerIterator = row.cellIterator();
 		while (headerIterator.hasNext()) {
 			Cell headerCell = headerIterator.next();
-			String headerValue = headerCell.getStringCellValue()
-					.trim();
+			CellType cellType = headerCell.getCellTypeEnum();
+
+			String headerValue;
+			if (cellType.equals(CellType.STRING)) {
+				headerValue = headerCell.getStringCellValue()
+						.trim();
+			} else {
+				headerValue = String.valueOf(headerCell.getNumericCellValue())
```

Comment:
If I upload a spreadsheet with a column header of `1`, this will end up setting the headerValue to `1.0`.

Is there another method in here to get the cell contents as a string even if it is numeric?

Reply:
The java excel parser assumes that any cell that has a number in it, must be a number cell.  Very frustrating.  Sorry I have not found a way around this, so that `*.0` will always be there since we have no way of knowing if that is what the user intended or not.

Reply:
Okay. I think this is acceptable (and preferable to not being able to load the spreadsheet at all).